frontend/src/pages/potions/[slug]/index.js

```diff
@@ -0,0 +1,77 @@
+import { getPotionsBySlug, getPotions } from '../../../lib/load_potions';
+
+import PotionDetails from '../../../components/pages/potions/[slug]/PotionDetails';
+import PotionMeta from '../../../components/pages/potions/[slug]/PotionMeta';
+
+const Potions = ({ data, links }) => {
+	const { attributes } = data;
+
+	return (
+		<>
+			<PotionMeta attributes={attributes} />
+			<PotionDetails attributes={attributes} apiLink={links?.self} />
+		</>
+	);
+};
+
+export async function getStaticPaths() {
+	const defaultSlugs = [
+		'ageing-potion',
+		'alihotsy-draught',
+		'amortentia',
+		'angel-s-trumpet-draught',
+		'antidote',
+		'antidote-to-common-poisons',
+		'antidote-to-uncommon-poisons',
+		'antidote-to-veritaserum',
+		'anti-paralysis-potion',
+		'armadillo-bile-mixture',
+		'babbling-beverage',
+		'baneberry-potion',
+		'baruffio-s-brain-elixir',
+		'beautification-potion',
+		'befuddlement-draught',
+		'blood-replenishing-potion',
+		'bloodroot-poison',
+		'bruise-removal-paste',
+		'bulgeye-potion',
+		'bundimun-pomade',
+	];
```

Comment:
That's not really needed here. I only used in the characters page to preload famous characters and to let search engines get those famous characters. Something like in the [movies page](https://github.com/danielschuster-muc/potter-db/blob/master/frontend/src/pages/movies/%5Bslug%5D/index.js#L16) is probably sufficient.